openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -585,14 +591,12 @@ def process_resources(self, instance, staging_dir, processors):
         resources = instance.data["resources"]
         color_management = lib.get_color_management_preferences()
 
-        # Temporary fix to NOT create hardlinks on windows machines
-        if platform.system().lower() == "windows":
-            self.log.info(
-                "Forcing copy instead of hardlink due to issues on Windows..."
-            )
-            force_copy = True
-        else:
-            force_copy = instance.data.get("forceCopy", False)
+        # Temporary disable all hardlinking, due to the feature not being used
```

Comment:
Can we add `TODO: ` for this comment so it's easily found in the codebase as code that might need some more work?